src/contrib/cluster/Akka.Cluster.Sharding.Tests.MultiNode/ClusterShardingFailureSpec.cs

```diff
@@ -43,7 +43,7 @@ protected ClusterShardingFailureSpecConfig(string mode)
                             ""System.Object"" = hyperion
                         }}
                     }}
-                    akka.loglevel = INFO
+                    akka.loglevel = WARNING
```

Comment:
So, instead of changing logging level per spec, could you try to configure logging level on the MultiNodeTestRunner side? So that all existing specs could still write logs as they want, and we did not have any restrictions on that for new specs - but MNTR would write only given log level to it's outputs.

This way we could easily configure that in future. For example, let it output DEBUG level when we are running / debugging new tests on local machine.

Reply:
That would also save you a lot of efforts from modifying other specs we have. Does it make sense?

Reply:
I guess that will also work. 
I'll take a look in the MNTR to see how it works and get back to you with specific questions if any.

Reply:
Sure, I'll be glad to help! Right now I do not have all the details from my head, but I remember that there are some actors that are handling arriving log events and producing output. So it should be a good place to inject filtering.

Reply:
[WIP PR](https://github.com/akkadotnet/akka.net/pull/4651)